src/Microsoft.Health.Dicom.SqlServer/Features/Store/SqlIndexDataStoreV1.cs

```diff
@@ -276,5 +276,60 @@ private async Task DeleteInstanceAsync(string studyInstanceUid, string seriesIns
                 }
             }
         }
+
+        public async Task<int> RetrieveNumDeletedExceedRetryCountAsync(int maxRetryCount, CancellationToken cancellationToken = default)
+        {
+            using (SqlConnectionWrapper sqlConnectionWrapper = await _sqlConnectionFactoryWrapper.ObtainSqlConnectionWrapperAsync(cancellationToken))
+            using (SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateSqlCommand())
+            {
+                sqlCommandWrapper.CommandText = @$"
```

Comment:
nit: while not much of a security risk as the only public parameter is an `int`, but should we be using `Parameters` instead?

Reply:
Changed to use paramaters